### PR TITLE
task/ICL-251 update resolver to v2

### PR DESCRIPTION
**Pull request by JGiter**

### CODE REVIEW

**Review by Harasz — APPROVED** [review]

File: src/chainConstants.ts [hunk 1]
```diff
@@ -32,3 +32,5 @@ export const EWC_DOMAIN_NOTIFER_ADDRESS =
 export const EWC_IDENTITY_MANAGER_ADDRESS =
   '0x2d1569f3a2006d21c0dc60eb13c8557b63ce5a8d';
 export const EWC_ADDRESS_1056 = '0xE29672f34e92b56C9169f9D485fFc8b9A136BCE4';
+export const EWC_RESOLVER_V2_ADDRESS =
+  '0xcf72f16Ab886776232bea2fcf3689761a0b74EfE';
```
Comment (jrhender) [inline src/chainConstants.ts:36]: @JGiter @Harasz I don't think this is correct for EWC: https://explorer.energyweb.org/address/0xcf72f16Ab886776232bea2fcf3689761a0b74EfE/transactions
Am I misunderstanding something?

Comment (Harasz) [reply src/chainConstants.ts:36]: Yeah, you're right. I didn't check it.

Comment (jrhender) [reply src/chainConstants.ts:36]: @JGiter shall we revert this change?

Comment (JGiter) [reply src/chainConstants.ts:36]: My bad. Fixing it in https://github.com/energywebfoundation/iam-contracts/pull/113

Comment (jrhender) [reply src/chainConstants.ts:36]: No worries, thanks

**Pull request merged**